prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md

```diff
@@ -0,0 +1,77 @@
+# How to Update the Observation Mapping Table
+
+## Introduction
+
+The observation mapping table maps a plethora of conditions, observations, and aoe question to codes in various code
+systems. The data for this table comes from multiple public sources. Once loaded in the database, the prime CLI can be
+used to poll public RESTful sources for updated data.
+
+A local csv file is kept both as a reference and to seed the local development environment. It should be updated
+periodically by dumping the table from staging/prod or using the CLI to update the local table.
+
+## Update prod/staging tables using RESTful sources
+TBD
+
+## Update the local CSV by getting tables from prod/staging
+Use the prime CLI `lookuptables get` command:  
+Change the `--env` value as needed
+```zsh
+prime lookuptables get -n observation-mapping -v 1 -o observation-mappings.csv --env prod
+```
+
+## Create or update prod/staging tables using a local CSV
+Use the prime CLI `lookuptables create` command:
+```zsh
+./prime lookuptables create prod -i ./observation-mapping.csv -a -n observation-mapping
```

Comment:
Just want to confirm that if this table already existed this is an update and doesn't drop what's there?

Reply:
Correct 👍 `create` handles that use case i.e. there is no separate `update` command.

```
Usage: prime lookuptables [OPTIONS] COMMAND [ARGS]...

  Manage lookup tables

Options:
  -h, --help  Show this message and exit

Commands:
  list             List the lookup tables
  get              Fetch the contents of a lookup table
  create           Create a new version of a lookup table
  activate         Activate a specific version of a lookup table
  diff             Generate a difference between two versions of a lookup
                   table
  loadall          Load all the tables stored as CSV in the specified
                   directory
  compare-mapping  Compares a sender compendium against an observation mapping
                   lookup table, outputting an annotated CSV
```

```
Usage: prime lookuptables create [OPTIONS]

  Create a new version of a lookup table

Options:
  -e, --env <name>       Connect to <name> environment. Choose between
                         [local|test|staging|prod]
  -i, --input-file PATH  Input CSV file with the table data
  -s, --silent           Do not generate diff or ask for confirmation
  -a, --activate         Activate the table upon creation
  -n, --name TEXT        The name of the table to perform the operation on
  --show-table           Always show the table to be created
  -f, --force            Force the creation of new table(s) even if it is
                         already exist
  -h, --help             Show this message and exit
```